### PR TITLE
Updated Microsoft Purview portal to Microsoft Purview compliance portal

### PR DESCRIPTION
**Pull request by robmazz**
@KCCross - merging 'Microsoft Purview portal' to 'Microsoft Purview compliance center' updates in ipcpub-purview project branch.

cc: @markjjo

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: exchange/exchange-ps/exchange/Add-ComplianceCaseMember.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ To add a member of an eDiscovery case, the user needs to be a member of the Revi
 - Create and edit compliance searches associated with a case.
 - Perform compliance actions (for example, export) on the results of a compliance search.
 
-To use this cmdlet in Security & Compliance Center PowerShell, you need to be assigned permissions. For more information, see [Permissions in the Microsoft Purview portal](https://docs.microsoft.com/microsoft-365/compliance/microsoft-365-compliance-center-permissions).
+To use this cmdlet in Security & Compliance Center PowerShell, you need to be assigned permissions. For more information, see [Permissions in the Microsoft Purview compliance portal](https://docs.microsoft.com/microsoft-365/compliance/microsoft-365-compliance-center-permissions).
```
[the remark this answers is not in the document]
Comment (JohanFreelancer9) [reply exchange/exchange-ps/exchange/Add-ComplianceCaseMember.md:39]: @robmazz, please make these changes throughout this PR. "Permissions in the Microsoft 365 compliance center" is the correct title of the link topic and the link should also be changed to a relative link. Many thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: exchange/exchange-ps/exchange/Add-ComplianceCaseMember.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (robmazz) [reply exchange/exchange-ps/exchange/Add-ComplianceCaseMember.md:39]: @JohanFreelancer9 - these are Purview branding changes slated for release on 4/19 and this is merge to our project branch

**Comment by robmazz** [comment]
@KCCross  - please merge these updates so we can review in the ipcpub-purview branch, thanks!

**Comment by KCCross** [comment]
merged.

**Comment by KCCross** [comment]
I've rolled main back into this too. make sure you pull down the latest ipcpub-purview.

**Pull request merged**